build-test.xml

```diff
@@ -15043,6 +15043,37 @@ url=${short.hostname}:8080</echo>
 		<start-docker-database />
 	</target>
 
+	<target name="start-docker-database-postgresql">
+		<stop-docker-database />
+
+		<if>
+			<not>
+				<isset property="database.version" />
+			</not>
+			<then>
+				<var name="database.version" value="14.4" />
+			</then>
+		</if>
+
+		<delete file="test.${user.name}.properties" />
```

Comment:
We should not be deleting custom files for testers, would be safer to append their file.